### PR TITLE
Hotdogs are no longer in the pastry crafting category. Moved to breads

### PR DESCRIPTION
**Pull request by Xoxeyos**
### Intent of your Pull Request

This moves hotdogs to the bread category.

### Why is this change good for the game?

Pastries are often confectioneries, and are used for deserts, a hotdog should not be eaten during desert, and it is not sweet enough to be a confectionery.

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
This will be the basis of the Wiki entry for your PR, and more information / detail is better for Wiki editors to integrate.

This literally just moves the hotdogs into the bread crafting category.

### What should players be aware of when it comes to the changes your PR is implementing?

Hotdogs have been moved in the crafting menu to breads.

### What general grouping does this PR fall under? 
Crafting changes.

### Are there any aspects of the PR that you would like us not to mention on the Wiki?

Nope.

:cl:  Xoxeyos
tweak: Hotdogs ARE NOT pastries. 
/:cl:

### CODE REVIEW

**Comment by Mqiib** [comment]
Hot dogs are sandwiches, please fix

**Comment by Xoxeyos** [comment]
> 
> 
> Hot dogs are sandwiches, please fix

https://discord.com/channels/134720091576205312/205784753352343552/833935126404268073

**Comment by Hopekz** [comment]
image
image

**Comment by Xoxeyos** [comment]
Atleast it wasn't a sandwich..

**Comment by Mqiib** [comment]
image
As you can see from this table, a hot dog would fall firmly in the "Ingredient purist/Structure neutral" plot on the middle left, firmly placing it in the realm of being a sandwich.

**Comment by Hopekz** [comment]
> Atleast it wasn't a sandwich..

https://strawpoll.com/s45j4sagw/r
NOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO

**Comment by Xoxeyos** [comment]
> 
> 
> > Atleast it wasn't a sandwich..
> 
> https://strawpoll.com/s45j4sagw/r
> NOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO

You clearly used inspect element, it's a s*ndwicher conspiracy.

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_burger.dm [hunk 1]
```diff
@@ -302,4 +302,15 @@
...
+
+/datum/crafting_recipe/food/hotdog
+	name = "Hot Dog"
+	reqs = list(
+		/datum/reagent/consumable/ketchup = 5,
+		/obj/item/reagent_containers/food/snacks/bun = 1,
+		/obj/item/reagent_containers/food/snacks/sausage = 1
+	)
+	result = /obj/item/reagent_containers/food/snacks/hotdog
+	subcategory = CAT_BREAD
```
Comment (SomeguyManperson) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_burger.dm]:
```suggestion
	subcategory = CAT_BURGER
```

**Comment by MegaEmpirical** [comment]
sandwich
